openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -333,7 +333,7 @@ This endpoint is used by a Credential Issuer that is already interacting with an
 
 ## Credential Offer {#credential-offer}
 
-The Credential Issuer sends Credential Offer using an HTTP GET request or an HTTP redirect to the Wallet's Credential Offer Endpoint defined in (#client-metadata).
+The Credential Issuer makes a Credential Offer by allowing the End-User to invoke the Wallet using the Wallet's Credential Offer Endpoint defined in (#client-metadata) (for example by clicking a link) and/or rendering a QR code containing the Credential Offer that the End-User can scan in a wallet or an arbitrary camera application.
```

Comment:
What about custom url schemes as clickable links?

Reply:
In that case the "Wallet's Credential Offer Endpoint" is a custom url, I think that case is covered?